base_layer/core/src/helpers/mod.rs

```diff
@@ -45,15 +46,21 @@ pub fn create_orphan_block(block_height: u64, transactions: Vec<Transaction>) ->
         .build()
 }
 
-pub fn create_mem_db() -> BlockchainDatabase<MemoryDatabase<HashDigest>> {
+pub fn create_mem_db(genesis_block: Option<Block>) -> BlockchainDatabase<MemoryDatabase<HashDigest>> /*,Block,UnblindedOutput)*/
+{
     let validators = Validators::new(
         MockValidator::new(true),
         MockValidator::new(true),
         MockValidator::new(true),
         MockValidator::new(true),
     );
+    let consensus_constants = match genesis_block {
+        Some(block) => ConsensusConstants::current_with_network(Network::LocalNet(Box::new(block))),
+        None => ConsensusConstants::current(),
```

Comment:
I think this needs to change to a new(Network type)

When we release mainnet, default will probably be mainnet

Reply:
The defaults seem to need some more work, let's fix it in a future PR.